source/plugin/text/templates.rst

```diff
@@ -0,0 +1,135 @@
+=======================
+Your first TextTemplate
```

Comment:
This appears in the index tree on the left, are you sure that this is what the title should be? It seems kind of inconsistent with the rest of the docs. Even if so, the entire thing should be in title case, Your First TextTemplate.